src/main/java/org/team1540/robot2024/commands/autos/AmpLanePABCSprint.java

```diff
@@ -0,0 +1,32 @@
+package org.team1540.robot2024.commands.autos;
+
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import org.team1540.robot2024.subsystems.drive.Drivetrain;
+import org.team1540.robot2024.subsystems.indexer.Indexer;
+import org.team1540.robot2024.subsystems.shooter.Shooter;
+import org.team1540.robot2024.util.AutoCommand;
+import org.team1540.robot2024.util.PathPlannerHelper;
+
+public class AmpLanePABCSprint extends AutoCommand {
+
+    public AmpLanePABCSprint(Drivetrain drivetrain, Shooter shooter, Indexer indexer){
+        super("AmpLanePABCSprint");
+
+        addPath(
+                new PathPlannerHelper(drivetrain, "AmpLanePABCSprint.1", true, true, true),
```

Comment:
is unclear what a PathPlannerHelper is based on this context (maybe a better name), why do they all need a drivetrain passed individually?